pkg/metrics/filters/filter_suite_test.go

```diff
@@ -0,0 +1,64 @@
+package filters
```

Comment:
These files probably should have the copyrights header

Not good that CI is not capturing it, but that's a separate issue :)